lib/handle-create-event.js

```diff
@@ -13,7 +14,7 @@ const template = readFileSync(pathResolve(__dirname, '..', 'instructions.md'), '
 
 async function handleCreateEvent (context) {
   const {ref, ref_type: refType, repository} = context.payload
-  const config = await context.config('first-timers.yml', {labels: ['first-timers-only', 'hacktoberfest'], repository: repository.name})
+  const config = await context.config('first-timers.yml', {labels: ['first-timers-only', 'hacktoberfest'], repository: repository.name, template: template})
```

Comment:
you can’t default `config.template` to `template` because it is the content of the template, not the path to the template file